.gitignore

```diff
@@ -19,6 +19,7 @@ node_modules
 .nuget/NuGet.exe
 project.lock.json
 .build
+.idea/
```

Comment:
Just curious, noticed this popping up in your PR's, what are you using IntelliJ for?

Reply:
Rider :)